src/index.html

```diff
@@ -6,9 +6,69 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#page_top">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="moyo"
+        >
```

Comment:
<img width="323" alt="Screenshot 2023-06-12 at 12 19 47" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/0fc5b494-2802-4360-978b-2688309cd4f9">
Currently, it renders strangely. Consider closing the self-closing image tag correctly.